### PR TITLE
make some internal globals const in TkBfield.cc

### PR DESCRIPTION
**Pull request by slava77**
online edit; after a compile check

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

File: MagneticField/ParametrizedEngine/src/TkBfield.cc [hunk 1]
```diff
@@ -11,18 +11,18 @@ using namespace magfieldparam;
 
 namespace {
 
-  BCylParam<float> fpar1{
+  const BCylParam<float> fpar1{
       4.90541f, 17.8768f, 2.02355f, 0.0210538f, 0.000321885f, 2.37511f, 0.00326725f, 2.07656f, 1.71879f};  // 2.0T-2G
-  BCylParam<float> fpar2{
+  const BCylParam<float> fpar2{
```
Comment (VinInn) [inline MagneticField/ParametrizedEngine/src/TkBfield.cc]: constexpr?

Comment (slava77) [reply MagneticField/ParametrizedEngine/src/TkBfield.cc]: makes sense; done.

**Comment by slava77** [comment]
@cmsbuild please test

File: MagneticField/ParametrizedEngine/src/TkBfield.cc (hunk 1 above)
Comment (slava77) [reply MagneticField/ParametrizedEngine/src/TkBfield.cc]: apparently clang disagrees

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/27422	cc0ab98
- technical

**Comment by fabiocos** [comment]
+1

**Pull request merged**